tests/Fixtures/LoadsFixtures.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Looks good but what is the value of changing this? It's gone from simple but readable data loading read to less simple and not so readable.

Reply:
Way easier to implement new tests and follows a standard paradigm.

Reply:
Sorted